### PR TITLE
Allow default values

### PR DESCRIPTION
**Pull request by ConfusedVorlon**
see previous discussion

### CODE REVIEW

**Comment by ConfusedVorlon** [comment]
this is looking even worse for number of commits. 
I thought I had done everything as a single commit on a new branch - but I guess my git-fu isn't up to the job.
Can you either live with this, or let me know how to fix? Git rebase -i seems to make things worse...

**Comment by ConfusedVorlon** [comment]
Ping.

**Comment by mfurtak** [comment]
Hi @ConfusedVorlon - I plan to review this on Thursday, 1/28. Thanks for your contribution and your patience!

**Comment by ConfusedVorlon** [comment]
@mfurtak excellent - thank you. I wondered what the 'assigned' meant.

File: README.md [hunk 1]
```diff
@@ -170,6 +170,51 @@ Before actually uploading anything to iTunes, ```deliver``` will generate a HTML
...
+Deliver has a special ```default``` language code which allows you to provide values that are not localised, and which will be used as defaults when you don’t provide a specific localised value.
+
+You can use this either in json within your deliverfile, or as a folder in your metadata file.
+
+Imagine that you have localised data for the following language codes:  ```en-US, de-DE, el, it```
+
+You can set the following in your deliverfile
+
+```
+release_notes({
```
Comment (KrauseFx) [inline README.md]: This should be without the `{`

Comment (ConfusedVorlon) [inline README.md]: what am I missing? There is a missing ' after the default, but this works, and I thought it was the correct syntax for a hash in ruby?
It's also the syntax you use for description in deliverfile.md

````
Localised values should be set like this

description({
  'en-US' => "English Description here",
  'de-DE' => "Deutsche Beschreibung hier"
})
```

**Comment by mfurtak** [comment]
Hi @ConfusedVorlon - This seems like great functionality to have! :+1: 

When fetching metadata from iTC for the first time, will these changes create a **default** directory if one doesn't exist? If not, do you think that might help the discoverability of this feature?

**Comment by ConfusedVorlon** [comment]
@mfurtak - nope, I haven't added any functionality to create the default directory.
I agree that would be a great thing to add. I'll leave that you you & @KrauseFx though!

**Comment by mfurtak** [comment]
@ConfusedVorlon Please check out #561 and let me know if it looks good to you!

**Comment by ConfusedVorlon** [comment]
hi - I didn't realise you were waiting for a response from me. I can't test it - but assume it is good. Looking forward to finally seeing this merged.

**Comment by mfurtak** [comment]
Great, will merge!

**Comment by ConfusedVorlon** [comment]
WooHoo. Thanks for reviewing and tidying up.

**Comment by mfurtak** [comment]
Sure thing - thanks for the contribution! :rocket:

**Pull request closed without merging**